openslides_backend/action/actions/user/create_update_permissions_mixin.py

```diff
@@ -0,0 +1,158 @@
+from typing import Any, Dict, List, Set, Tuple, cast
+
+from ....permissions.permission_helper import has_perm
+from ....permissions.permissions import (
+    CommitteeManager,
+    OrganisationManagementLevel,
+    Permission,
+    Permissions,
+)
+from ....services.datastore.commands import GetManyRequest
+from ....shared.exceptions import PermissionDenied
+from ....shared.patterns import Collection, FullQualifiedId
+from ...action import Action
+
+
+class CreateUpdatePermissionsMixin(Action):
+    field_rights: Dict[str, list] = {
```

Comment:
I don't like the repetitivity of this. How do you feel about inversing this hardcoded dictionary (so we have a `Dict[List[Union[Permission, OrganisationManagementLevel, CommitteeManager]], List[str]]) and calculating the inverse of that in the `__init__` method?

Reply:
The repetitivity is really ugly, but
- easier to maintain
- less error-prone, because of less code
- needs less time on startup